javascriptv3/example_code/lambda/README.md

```diff
@@ -1,30 +1,63 @@
-# AWS Lambda JavaScript SDK v3 code examples
-AWS Lambda lets you run code without provisioning or managing servers. You pay only for the compute time you consume.
+# AWS Lambda code examples for the AWS SDK for JavaScript in Node.js
+
+## Overview
+
+Shows how to use the SDK for JavaScript in Node.js with AWS Lambda (Lambda) to manage and invoke
+functions.
+
+AWS Lambda is a serverless, event-driven compute service that lets you run code for virtually any type of application or backend service without provisioning or managing servers. You can trigger Lambda from over 200 AWS services and software as a service (SaaS) applications, and only pay for what you use.
+
+## ⚠️ Important
+
+- Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
 ## Code examples
-This is a workspace where you can find the following AWS SDK for JavaScript v3 AWS Lambda examples. 
-- Create a table using Lambda function. See [documentation](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/lambda-create-table-example.html).
 
-**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
-[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-example-javascript-syntax.html).
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Create a Lambda function](./actions/create-function.js)(`CreateFunction`)
+- [Delete a Lambda function](./actions/delete-function.js)(`DeleteFunction`)
+- [Get a Lambda function](./actions/get-function.js)(`GetFunction`)
+- [Invoke a Lambda function](./actions/invoke.js)(`Invoke`)
+- [Update Lambda function code](./actions/update-function-code.js)(`UpdateFunctionCode`)
+- [Update Lambda function configuration](./actions/update-function-configuration.js)(`CreateFunction`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Interactive Lambda command line tool](./scenarios/basic)
```

Comment:
This title must match the `title_abbrev`: [Get started with functions](https://docs.aws.amazon.com/code-library/latest/ug/lambda_code_examples_scenarios.html)

Reply:
Fixed.